action.yml

```diff
@@ -24,5 +28,30 @@ inputs:
     default: 'gpt-4-turbo-preview'
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile' 
\ No newline at end of file
+  using: 'composite'
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+        cache: 'pip'
+    
+    - name: Install dependencies
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ${{ github.action_path }}/requirements.txt
+
+    - name: Run OtterAI Code Review
+      shell: bash
```

Comment:
🤖 The 'Run OtterAI Code Review' step has been added, which is the core functionality of the action. Ensure that the environment variables are properly set and validated before running the Python script.